package.json

```diff
@@ -60,7 +60,7 @@
     "license-check": "jsgl --local ./",
     "lint": "yarn eslint && yarn lint-check && yarn pretty-check && yarn lint-cspell",
     "lint-check": "eslint-config-prettier .eslintrc.js",
-    "lint-cspell": "cspell --unique",
+    "lint-cspell": "cspell --unique --gitignore --no-progress --no-must-find-files",
```

Comment:
Gitignore already specified in config file https://github.com/graphql/graphiql/blob/main/cspell.json